internal/command/quota_period.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/zitadel/zitadel/internal/repository/quota"
 )
 
+// TODO: why is a query implemented on the command side?
```

Comment:
@eliobischof we should discuss this on monday

Reply:
Because we need that data in order to produce quota notified events.
We didn't find a good reference implementation so we decided to die this death.
I think that's what you and @livio-a and I aggreed upon?

Reply:
It's basically a write model query, so where is it put best? queries? commands?

Reply:
This call is made every time a call hits the api, I would prefer to do the query as of system time because if not the users will block each other

Reply:
Implemented this as discussed: https://github.com/zitadel/zitadel/pull/5306